src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -535,6 +535,60 @@ An GoogleComputeEngine NodeSource can now be added using the *Create Node Source
+- *Azure privileges:* If your account does not have full Admin privileges on your subscription, please make sure you have rights to:
+* Access a https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups[resource group] of your choice.
+* Create VMs in that resource group.
+* Delete VMs in that resource group.
+- *Service Principal credentials:* ProActive uses Azure's API to manage the VMs in the infrastructure. This API requires a set of Active Directory _service principal credentials_ for non interactive login (client, secret, tenant). If you do not have them, you can create them via the https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal[Azure Portal] or https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest[Azure CLI]. Please ask your administrator in case you do not have the privileges to create them. The credentials should be strings of hexadecimal values like _1a2b3c4d-5e6f-1a2b-3c4d-5e6f-1a2b3c4d5e6f_.
+- *Custom Azure +image+:* ProActive will create the infrastructure's VMs from an Azure image. You will need to have a custom linux image available on the resource group of your choice. If you do not have one, you can create it by following https://docs.microsoft.com/en-us/azure/virtual-machines/linux/tutorial-custom-images[this tutorial].
+
+===== Infrastructure Configuration
+
+Once all the pre-requisites are met, you are ready to create an Azure infrastructure. On the ProActive's Resource Manager portal, select _AzureInfrastructure_ from the Infrastructure drop-down list.
+
+The following fields are available to configure your Azure infrastructure.
+
+- *clientId:* Corresponds to the Service Principal _application ID_.
+- *secret:* Corresponds to the _secret_ value (password) of the Service Principal.
+- *domain:* Corresponds to the Azure _domain_, _tenant ID_, or _directory ID_ of your Active Directory.
+- *subscriptionId:* The _ID_ of your Azure subscription.
+
+The next four parameters are optional, and apply to advanced Azure endpoint configurations only.
+
+- *authenticationEndpoint:* Azure Authentication endpoint.
+- *managementEndpoint:* Azure Management endpoint.
+- *resourceManagerEndpoint:* Azure Resource Manager endpoint.
+- *graphEndpoint:* Azure AD Graph endpoint.
+
++* * * * *+
+
+- *rmHttpUrl:* Here you should provide your server's URL or public IP (e.g. try.activeeon.com). A default value will be generated from the system properties but you might need to modify it according to your network configuration so that it is accessible from your Azure nodes.
+- *connectorIaasUrl:* The URL of your _Connector IaaS_ service. A default value will be generated as well (e.g. +http://try.activeeon.com:8080/connector-iaas+).
+- *image:* The name or id of your custom linux image. The image should be located within the provided subscription and accessible to the service principal.
```

Comment:
I'm not an Azure expert, but maybe image should be located in the specified region as well.

Reply:
You are right, I have seen that error. Thanks for noticing!